api/pyproject.toml

```diff
@@ -66,3 +66,109 @@ MOCK_SWITCH = "true"
 CODE_MAX_STRING_LENGTH = "80000"
 CODE_EXECUTION_ENDPOINT="http://127.0.0.1:8194"
 CODE_EXECUTION_API_KEY="dify-sandbox"
+
+
+[tool.poetry]
+name = "dify-api"
+version = "0.6.8"
```

Comment:
```suggestion
version = "0.6.10"
```

Reply:
Yep. It's best to update to the latest dependencies.

Reply:
fixed

Reply:
Thanks.